### DISCUSSION TITLE
sidebar/sanfene/collection

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#83] [Announcements]
# sidebar/sanfene/collection

图文详解 30 道Java集合框架面试高频题，这次吊打面试官，我觉得稳了（手动 dog）

https://tobebetterjavaer.com/sidebar/sanfene/collection.html

**Comment by LVXUEYANGTiger** [comment]
看完

**Comment by HelloDam** [comment]
爱了爱了

**Comment by a894985555** [comment]
"注意，在链表中插入节点的时候，如果链表长度大于等于 8，则需要把链表转换为红黑树"。应该是链表长度大于8

**Comment by 2244721984** [comment]
如果元素重复，HashSet不会覆盖值，而是拒绝添加重复元素，原集合元素保持不变吧。

**Comment by Xiamai** [comment]
16题的图画错了吧，移位运算不太对
